pkg/operator/controller.go

```diff
@@ -116,7 +120,7 @@ type Controller struct {
 	clients    *regopclient.Clients
 }
 
-func (c *Controller) createOrUpdateResources(cr *imageregistryv1.Config) error {
+func (c *Controller) createOrUpdateResources(cr *imageregistryv1.Config, pcr *imageregistryv1.ImagePruner) error {
```

Comment:
There is no need in this change.

Reply:
removed in next push, that was a leftover that i didn't catch from the previous way i was doing things.